mesonbuild/compilers/compilers.py

```diff
@@ -669,6 +670,9 @@ def get_option_compile_args(self, options):
     def get_option_link_args(self, options):
         return []
 
+    def get_default_include_dirs(self):
+        return []
```

Comment:
This should probably go on `CCompiler`, same as `get_library_dirs()`.

Reply:
Ok